docs/editor/profiles.md

```diff
@@ -128,6 +128,211 @@ Profiles can be used to customize VS Code for students to ease the use in a clas
+* [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) - Your AI pair programmer.
+
+This profile also sets the following settings:
+
+```json
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter",
+        "editor.formatOnType": true,
+        "editor.formatOnSave": true
+    },
+    "editor.inlineSuggest.enabled": true,
+    "editor.lineHeight": 17,
+    "breadcrumbs.enabled": false,
+    "files.autoSave": "afterDelay",
+    "jupyter.themeMatplotlibPlots": true,
+    "jupyter.widgetScriptSources": [
+        "unpkg.com",
+        "jsdelivr.com"
+    ],
+    "notebook.experimental.outputScrolling": true,
+    // "notebook.outline.showCodeCells": true,
+    "workbench.colorTheme": "Default Dark+ Experimental",
+    "files.exclude": {
+        "**/.csv": true,
+        "**/.parquet": true,
+        "**/.pkl": true,
+        "**/.xls": true
+    }
+```
+
+### Doc Writer Profile Template
+
+The Doc Writer profile is a good lightweight setup for writing documentation. It comes with the following extensions:
+* [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) - Spelling checker for source code.
+* [GitHub Markdown Preview](https://marketplace.visualstudio.com/items?itemName=bierner.github-markdown-preview) - Changes VS Code's built-in markdown preview to match GitHub.
+* [Markdown Checkboxes](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-checkbox) - Adds checkbox support to the built-in markdown preview.
+* [Markdown Emoji](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-emoji) - Adds emoji syntax support to VS Code's built-in markdown preview and markdown cells in notebook
+* [Markdown Footnotes](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-footnotes) - Adds ^footnote syntax support to VS Code's built-in markdown preview.
+* [Markdown Preview Github Styling](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles) - Changes VS Code's built-in markdown preview to match Github's style.
+* [Markdown Preview Mermaid Support](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid) - Adds Mermaid diagram and flowchart support to VS Code's builtin markdown preview.
+* [Markdown yaml Preamble](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-yaml-preamble) - Renders yaml front matter as a table in the built-in markdown preview.
+* [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) - Markdown linting and style checking for Visual Studio Code.
+* [Word Count](https://marketplace.visualstudio.com/items?itemName=ms-vscode.wordcount) - Markdown Word Count Example - a status bar contribution that reports out the number of works in a Markdown document as you interact with it.
```

Comment:
Hmm, the Word Count extension hasn't been updated since 2015 and only has 117K installs. I wonder if there is a better extension that includes this functionality?

Reply:
We can remove it? What do you think?